### ISSUE TITLE
Decrypt with openssl , or other cli tool?

### ISSUE THREAD

**Issue by smoser** [Fourmilab/JavaScrypt_browser_based_cryptography#1]
tldr; How can I decrypt data encrypted with javascrypt using openssl or some other program.

I have some content that is encrypted with Javascrypt via dokuwiki plugin [dokucrypt](https://www.dokuwiki.org/plugin:dokucrypt).

Content looks like this, and it can be decrypted at https://www.fourmilab.ch/javascrypt/javascrypt.html with the text password 'secret'
```
#####  Encrypted: decrypt with http://www.fourmilab.ch/javascrypt/
?b64Py5y9JZMVa728IXwqaBh+f9Tbz9jP9OvCY2UdexIz5i6bLRsCOF2IA9qECpp
Bc6oY+BAihtLdzyrtQA3x6yrxg==?64b
#####  End encrypted message
```

I'd like to decrypt that block of data using 'openssl' or some other tool that does aes-256 encryption.
I  admit to not having tried very hard to find this out myself.  I tried base64 decode the content between ?b64 and ?64b and then decrypt it with `openssl enc -d -aes-256-cbc -in /tmp/enc.txt` but that failed with 'bad magic number'.

This seems like it would be a good addition to README.md that I'd be happy to provide a PR for.
